cypress.json

```diff
@@ -0,0 +1,25 @@
+{
+  "baseUrl": "http://localhost:3000",
+  "experimentalSessionAndOrigin": true,
+  "video": false,
+  "viewportHeight": 1080,
+  "viewportWidth": 1920,
+  "defaultCommandTimeout" : 10000,
+  "component": {
+    "componentFolder": "src",
+    "testFiles": "**/*spec.tsx"
+  },
+  "env": {
+    "DB_HOST": "127.0.0.1",
+    "DB_USER": "root",
+    "DB_PASSWORD": "",
+    "DB_PORT": 3306,
+    "BACKEND_URL": "http://localhost/toolbox-backend/public/",
+    "APP_CLIENT_ID": 2,
+    "TEST_LOGIN": {
+      "username": "max@test.test",
+      "password": "password"
```

Comment:
Put this into two env variables, a.e. "TEST_LOGIN_USERNAME, TEST_LOGIN_PASSWORD"